### ISSUE TITLE
Datepicker uses current date rather than the date of the date from the model

### ISSUE THREAD

**Issue by wartab** [nkalinov/ng2-datetime#30]
I am trying to display a form containing a datetime input. Since I want to reload the state of a previously filled in form, I prefill the form using the date the user has entered the last time he filled in that form. The datepicker does show the current date instead of the initial date from the model, the timepicker shows the correct time, though.

The template is the following:

```html
<div class="form-group">
    <label for="earliestDeparture" class="control-label">{{'earliestDeparture'}}</label>
    <datetime [(ngModel)]="model.earliestDeparture"
        [timepicker]="{showMeridian: false, minuteStep: 1}"
        [datepicker]="{autoclose: true, todayBtn: 'linked', todayHighlight:true, assumeNearbyYear: true, format: 'D, d MM yyyy'}"
     name="earliestDeparture"></datetime>
    {{model.earliestDeparture}}
</div>
```
For debugging purposes, I displayed the model value as plain text and as you can see, the model contains the correct date. Despite that, the datepicker does not, it shows the current date instead.

![9fb49a0b95b7ab003280ea365cfd9db1](https://cloud.githubusercontent.com/assets/9607577/17103385/58c34518-527e-11e6-81f6-d0aae461ab54.png)

I tinkered around and found out that if you replace `this.datepicker.datepicker('update', date.toUTCString());` by `this.datepicker.datepicker('update', date);`, the correct date shows (transmitting the Date object rather than the UTC string representation of that date).

If you have further questions about how to reproduce it, don't hesitate to ask.

**Comment by rafalkasa** [comment]
Hi @wartab

It's know issue I created pull request because of that.
I had conversation with @nkalinov and he has a plan to merge my changes and soon release the new 1.1.4 version of ng2-datetime.

**Comment by wartab** [comment]
Ah sorry, I saw your pull request, but I thought it was just about making it compatible with @angluar/forms (I already adapted that part from you.).

Thanks for helping out :+1:

**Comment by wartab** [comment]
I looked at your pull request in that regard and there are other instances where the UTC string is given to `datepicker("update", ...)`. Is it not necessary to change that too?

**Comment by rafalkasa** [comment]
Good point I didn't check this places because, I had problems only with loaded date from Json or when I try to set initial date other than today.

**Comment by nkalinov** [comment]
closed / c7d8682df4837e578249df0c9f32f66c17ac28ba
Version bumped to 1.1.4
